e2e/tests/smoke/smoke_test_helper.go

```diff
@@ -0,0 +1,416 @@
+package smoke
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/clients"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+	"github.com/rook/rook/e2e/rook-test-framework/enums"
+	"github.com/rook/rook/e2e/rook-test-framework/utils"
+)
+
+type SmokeTestHelper struct {
```

Comment:
the golang naming convention is that you don't have a type prefix with the same name as the package. Since you're in the smoke package, this type would just be called TestHelper